scipy/signal/signaltools.py

```diff
@@ -2645,11 +2642,16 @@ def decimate(x, q, n=None, ftype='iir', axis=-1):
     q : int
         The downsampling factor.
     n : int, optional
-        The order of the filter (1 less than the length for 'fir').
-    ftype : str {'iir', 'fir'}, optional
-        The type of the lowpass filter.
+        The order of the filter (1 less than the length for 'fir'). Defaults to
+        8 for 'iir' and 30 for 'fir'.
+    ftype : str {'iir', 'fir'} or ``lti`` instance, optional
+        If 'iir' or 'fir', specifies the type of lowpass filter. If an instance
+        of an ``lti`` object, uses that object to filter before downsampling.
     axis : int, optional
         The axis along which to decimate.
+    zero_phase : bool
+        Prevent phase shit by filtering with ``fitfilt`` instead of
```

Comment:
Besides the typo, if you surround function names in the current namespace with single backticks, they get turned into links in the html docs. So goes for ``filtfilt`` here and ``lti`` above.